### PR TITLE
fix: adjust seedlot details page

### PR DESCRIPTION
**Pull request by mgaseta**
# Description
Closes #NoIssue

Adjusted the buttons labels and when to display them, according with the seedlot status.
Fixed breadcrumbs for TSC admins.

#### New
- Created new utils function for seedlot pages breadcrumbs.

#### Changed
- Permissions to display buttons labels on seedlot details page.

#### Removed
- Repeated breadcrumbs functions.

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [ ] 🤖 Added tests

###  What gif/image best describes this PR or how it makes you feel?

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-spar-27-frontend.apps.silver.devops.gov.bc.ca/)
- [Backend](https://nr-spar-1527-backend.apps.silver.devops.gov.bc.ca/swagger-ui/index.html)
- [Oracle-API](https://nr-spar-1527-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-spar/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge.yml)

### CODE REVIEW

File: frontend/src/shared-constants/shared-constants.ts [hunk 1]
```diff
@@ -21,3 +21,5 @@ export const LARGE_SCREEN_WIDTH = 1056;
 export const TSC_ADMIN_ROLE = 'SPAR_TSC_ADMIN';
 
 export const PLACE_HOLDER = '--';
+
+export const MINISTRY_OF_FOREST_ID = '00012797';
```
Comment (craigyu) [inline frontend/src/shared-constants/shared-constants.ts:25]: Nice, could you update other places that uses this ID too?

I found them in:
- frontend/src/contexts/AuthProvider.tsx
- frontend/src/views/Seedlot/ContextContainerClassA/constants.tsx

**Review by craigyu — APPROVED** [review]

File: frontend/src/shared-constants/shared-constants.ts (hunk 1 above)
Comment (mgaseta) [reply frontend/src/shared-constants/shared-constants.ts:25]: Fixed here: https://github.com/bcgov/nr-spar/pull/1527/commits/e2ec8784c32bdc3a7a6b30feec9bdc7ca3d23f45

**Review by RMCampos — APPROVED** [review]
LGTM

**Comment by RMCampos** [comment]
Bypassing failing Cypress tests. As discussed in the Teams Chat, a possible fix will be addressed shortly, taking in acount promise-like and callbacks inside cypress assertions (probably the root-cause of the issue).

**Pull request merged**